src/backend/auth.api.ts

```diff
@@ -47,12 +47,15 @@ const register = async (userData: {
       throw new Error("User data can't be saved properly");
     }
 
-    const createVerify = await account.createVerification(
-      `${process.env.NEXT_PUBLIC_BASE_URL}/verify`,
-    );
-
-    if (!createVerify) {
-      throw new Error("Error sending verification email");
+    // send verification email in production mode
+    if (process.env.NODE_ENV === "production") {
```

Comment:
Mention about node_env in .env.example

Reply:
@Sanchitbajaj02 I have not added any extra env variables. `NODE_ENV` is a standard environment variable provided by node environment.

![Screenshot_20240724-003754](https://github.com/user-attachments/assets/19aba888-636b-42a8-b7b8-fa1cf17d365d)